README.md

```diff
@@ -135,30 +135,6 @@ NavigationViewController(for: route, styles: [CustomStyle()])
 * `navigationViewController:didRerouteAlongRoute:`: Fired as soon as the SDK receives a new route.
 * `navigationViewController:didFailToRerouteWithError:`: Fired when SDK receives an error instead of a new route.
 
-## Building your own custom navigation UI
-
-If you need additional flexibility, you can use the following building blocks to build your own custom navigation UI:
-
-* [Interactive map SDK for iOS](https://www.mapbox.com/ios-sdk/) and [macOS](https://mapbox.github.io/mapbox-gl-native/macos/)
-* [Mapbox Studio](https://www.mapbox.com/studio/)
-  * Design custom maps with live traffic overlays
```

Comment:
I think the readme should continue to link to the map SDK and Studio in some capacity, since we do want applications to use the map SDK for all their map needs, not just navigation.

Reply:
@1ec5 let's find a new home for this language somewhere else then. It doesn't make sense here.